actions/image_partition_action.go

```diff
@@ -469,6 +495,12 @@ func (i ImagePartitionAction) Run(context *debos.DebosContext) error {
 			}
 		}
 
+		if p.PartUUID != "" {
```

Comment:
May be better to check if len > 0

Reply:
Also check if it's gpt

Reply:
I think  no need to check `gpt` here because  `gpt` is  checked when validate  `partUUID`  in `Verify()` function

Reply:
yeah, it was more so we don't get confused in future.

Maybe a comment as a reminder would be OK ? Like: `/* PartUUID will only be set for gpt partitions */`

Reply:
Yes, I will do that.